pkg/vm/interop.go

```diff
@@ -107,3 +125,128 @@ func init() {
 		return defaultVMInterops[i].ID < defaultVMInterops[j].ID
 	})
 }
+
+// EnumeratorCreate handles syscall Runtime.Enumerator.Create.
```

Comment:
I think the comments are wrong here, these syscalls are in `Neo` namespace, not in `Runtime`.